backend/src/main/java/com/MerchStore/backend/Model/Order.java

```diff
@@ -5,15 +5,15 @@
 public class Order {
     private final long orderId;
 
-    private int quantity;
+    private long quantity;
 
     private double totalAmount;
 
     private String orderStatus;
 
     private final long cartId;
 
-    public Order(long orderId, int quantity, double totalAmount, String orderStatus, long cartId){
+    public Order(long orderId, long quantity, double totalAmount, String orderStatus, long cartId){
```

Comment:
I dont think someone would order a large amount of any given product, int was just fine!

Reply:
I have changed the data type of quantity to int.